src/components/Card.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useContext, useEffect } from 'react'
+import { MyCollectionContext, isCollected } from 'utils'
+
+type CardProps = {
+  cardData: PokemonCard
+}
+
+const Card = ({ cardData }: CardProps) => {
+  const { dispatch, myCollection } = useContext(MyCollectionContext)
```

Comment:
I would use a custom hook instead of using straight context everywhere: https://kentcdodds.com/blog/how-to-use-react-context-effectively